cxx4opencl/intro.txt

```diff
@@ -7,13 +7,22 @@
 
 This language is built on top of OpenCL C v2.0 and {cpp}17 enabling
 most of regular {cpp} features in OpenCL kernel code. Most functionality
-from OpenCL C and {cpp} is inherited. Restrictions are documented
-in <<restrictions, _Restrictions_>> sections.
+from OpenCL C and {cpp} is inherited. The following two sections of this
```

Comment:
```This document includes a description of <<diff2openclc, _the differences to OpenCL C_>> and <<diff2cxx, _the differences to {cpp}_>>.```

Please also make sure the lines are wrapped at 80 columns.

Reply:
Hmm but it is only 2 sections that explain the difference not the entire document. What is it that you want to be changed here exactly?

Reply:
I find "the following two sections" a bit ambiguous, as the reader may also think that it refers to the next two paragraphs.

Reply:
Ok, I provided more description now.